tests/cpptests/test_cpp_query.cpp

```diff
@@ -526,7 +526,7 @@ TEST_F(QueryTest, testParser_v2) {
   const char *qt = "(hello|world) and \"another world\" (foo is bar) -(baz boo*)";
   QASTCXX ast;
   ast.setContext(&ctx);
-  ASSERT_TRUE(ast.parse(qt));
+  ASSERT_TRUE(ast.parse(qt, version));
```

Comment:
How was it tested up til now?

Reply:
With the default dialect 1 (which was an error)